### PR TITLE
Channels intended usage: WS broadcast of on-chain txs

### PR DESCRIPTION
**Pull request by velzevur**
Addressing changes introduced by aeternity/epoch#1153

### CODE REVIEW

File: epoch/api/channels_api_usage.md [hunk 1]
```diff
@@ -173,18 +173,26 @@ The initiator receives the following message
  }
 ```
 
+#### Signed channel_create_tx
+Both participans receive the co-signed `channel_create_tx`:
```
Comment (uwiger) [inline epoch/api/channels_api_usage.md]: participans -> participants

File: epoch/api/channels_api_usage.md [hunk 2]
```diff
@@ -173,18 +173,26 @@ The initiator receives the following message
...
+```
+{'action': 'on_chain_tx',
+ 'payload': {'tx': 'tx$9mmqY8QRoYS8QR5nyRkWwTyhwecE8NzHotRK1yMjvfvyPfsaBNwWFDrmzG3M5rHKARm39AqeJpgN4Aj7V57yP4KusptAvMA2r3y593vr51ubXN8zo8t8qg7Arb32wWhmYDvRXVGKhfYoqu91BpkammeY69xCjGuQrXKDk3t9aPCcrvM3PbERAMvkmvD9dsb2H5iujGin8qZTy42xsUYS6QcTiaomxo8CmieGSvZkAai1KCZ84MZzsAjsR1FP54Su7rfBPXZTd5u2AmyF7LwZoHQsJEspLnzXxun2MtKTkhR3du8PY'
+            }
+}
+```
+Using its hash participants can track its progress on the chain - entering the mempool, block inclusion and a number of confirmations.
```
Comment (uwiger) [inline epoch/api/channels_api_usage.md]: "Using its hash, participants can track its progress on the chain: entering the mempool, block inclusion ..." (comma after "hash", replace "-" with ":")

File: epoch/api/channels_api_usage.md [hunk 3]
```diff
@@ -173,18 +173,26 @@ The initiator receives the following message
...
+channel's software to the node and goes to the mempool. Having calculated its hash, one can validate it using the HTTP API:
 ```
 $ curl 'http://localhost:3013/v2/tx/th$hNyHzj4dSzyBqReAMR36GGz1mhuXxQFuES3AnPkXkuY2w6dZb?tx_encoding=json'
 ```
 if the `block_hash` is `none` - then the transaction is still in the mempool.
 
 ### Block inclusion
-When the transaction is included in a block - this is the first confirmation. A block height timer
+When the transaction is included in a block - this is the first confirmation. A block height timer is
 started and it ends after `minimum_depth + 1` confirmations. Default value for
```
Comment (uwiger) [inline epoch/api/channels_api_usage.md]: Strictly speaking, isn't it `minimum_depth` confirmations after the block that includes the `create_tx`?

File: epoch/api/channels_api_usage.md [hunk 4]
```diff
@@ -451,9 +459,18 @@ it and then post it back via a WebSocket message:
 }
 ```
 
+#### Signed channel_close_mutual_tx
+Both participans receive the co-signed `channel_close_mutual_tx`:
```
Comment (uwiger) [inline epoch/api/channels_api_usage.md]: participants

File: epoch/api/channels_api_usage.md [hunk 5]
```diff
@@ -451,9 +459,18 @@ it and then post it back via a WebSocket message:
...
+```
+{'action': 'on_chain_tx',
+ 'payload': {'tx': 'tx$T7f9gwgv5SsoFi3uoKcjZcfG9wXqS52nRHiCZccmc2pYAxc17sfDDNu6EAVgxQQxD2pvbQGBBKpatdkczSQWPxqMdXpgPgKEqEGxzmqJmeNMFR81KotMjPAiJciuinTQ1Gr2jLnfck3mGhnpXT9DrniEYdqnd53nP2XGMPsodRC8dUUMFzANh3YzBn3xsrNA1eHVBu4SZVMGhUHR3baVGKNpdHhsz8RiYTc6bjHBDhRiRPtgfvF4swJGDUsgWpAciLnssdq8AZ9cVy5JyReg3mf8UJaSLfvVUATyN9LN3Jf3pW'
+            }
+}
+```
+Using its hash participants can track its progress on the chain - entering the mempool, block inclusion and a number of confirmations.
```
Comment (uwiger) [inline epoch/api/channels_api_usage.md]: "Using its hash, participants can track its progress on the chain: entering the mempool, block inclusion ..."

File: epoch/api/channels_api_usage.md [hunk 6]
```diff
@@ -451,9 +459,18 @@ it and then post it back via a WebSocket message:
...
+
 ### Channel closing
-After both parties have co-signed the `channel_close_mutual_tx` the
-transaction is posted on the chain and the microservice handling the off-chain
+After both parties have received the co-signed the `channel_close_mutual_tx` transaction, it is posted on the chain and the microservice handling the off-chain
```
Comment (uwiger) [inline epoch/api/channels_api_usage.md]: "the co-signed `channel_close_mutual_tx`" (one "the" too many)

File: epoch/api/channels_api_usage.md (hunk 3 above)
Comment (velzevur) [reply epoch/api/channels_api_usage.md]: As discussed, this is fine

**Pull request merged**